### ISSUE TITLE
Feature Request - Disabled State

### ISSUE THREAD

**Issue by LittleTal** [wix/wix-style-react#2368]
## The request
Disabled state in all components should be changed according to a new design.

Motivation: currently disabled state colors are not accessible- the contrast ratio is very low.

New disabled colors mapping:
https://docs.google.com/spreadsheets/d/1_Q2-zui1EwPqQ3JzE0zgevmIfTbEIbojgsbbz8h5aF8/edit?usp=sharing

Note that new colors are not Hex colors, it's `D10` and `D80` with an alpha channel.
It works well on a colored background. 

Example of components with new disables colors in Zeplin:
https://zpl.io/VQ994An

@shlomitc What do you think?

**Comment by shlomitc** [comment]
mostly done
